src/View.elm

```diff
@@ -69,38 +52,91 @@ statusContentView status =
                     "spoiler" ++ (toString status.id)
             in
                 div [ class "status-text spoiled" ]
-                    [ div [ class "spoiler" ] <| formatContent status.spoiler_text
+                    [ div [ class "spoiler" ] <| (ViewHelper.formatContent status.spoiler_text status.mentions)
```

Comment:
Nit: I realize `spoiler_text` doesn't actually contain HTML, but plain text it seems. No big deal.